### PR TITLE
Card components

### PR DESCRIPTION
**Pull request by GarvG**
Broke my PR it has only Card-component

### CODE REVIEW

**Review by iRohitGaur — CHANGES_REQUESTED** [review]
Don’t put image of code. User wont be able to copy the code.

File: card.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                    style="width: 1162px; height: 598px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                  
+                </div>
+
+                <div class="center-container">
+                    <h3 class="headers">HORIZANTAL CARD WITH BADGES!</h3>
```
Comment (iRohitGaur) [inline card.html]: Horizontal*

File: card.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    opacity: 0.6;
+}
+.card-TEXT
+{
+    background-color: white;
+    width: 330px;
+    height: 320px;
+    transition: 0.8s;
+    border-radius: 5px;
+    margin-bottom: 20px;
```
Comment (iRohitGaur) [inline card.css:175]: use rem

File: card.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    position: relative;
+    top: -60px;
+    left: 100px;
+    font-size: 24px;
+    font-weight: 700;
+    background-color: black;
+    color: white;
+    border: none;
+    border-radius: 6px;
+    width: 170px;
```
Comment (iRohitGaur) [inline card.css:162]: use rem

**Review by iRohitGaur — APPROVED** [review]
Not using rem at all. Everything is mentioned in px. Inconsitency in class spacing. Please update if possible. Merge with updates.

File: card.css [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    background-color: red;
+    font-size: 16px;
+    font-weight: 800;
+    padding: 2px;
+    border-radius: 6px;
+}
+.card-1
+{
+    height: 150px;
+    width: 330px;
```
Comment (zoef-yt) [inline card.css]: use relative units

**Review by zoef-yt — APPROVED** [review]
Use relative units such as rem. and remove the code images if you are not using them anymore

File: card.html (hunk 1 above)
Comment (GarvG) [reply card.html]: Changed!

File: card.css (hunk 4 above)
Comment (GarvG) [reply card.css]: done with rem

File: card.css (hunk 3 above)
Comment (GarvG) [reply card.css:162]: okay

File: card.css (hunk 2 above)
Comment (GarvG) [reply card.css:175]: okay

**Pull request merged**